docs/installation.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html>
+<head>
+<link rel="stylesheet" type="text/css" href="cbat.css">
+<title>CBAT Installation</title>
+</head>
+<body>
+
+<h1>CBAT Installation</h1>
+
+<section>
+
+<p>Table of contents:
+
+<ul>
+  <li><a href="#docker">Docker Installation</a></li>
+  <li><a href="#manual">Manual Installation</a></li>
+</ul>
+</section>
+
+<section>
+
+<a name="docker"></a>
+<h2>Docker Installation</h2>
+
+<p>Coming soon.
+
+</section>
+
+<section>
+
+<a name="manual"></a>
+<h2>Manual Installation</h2>
+
+<h3>Prerequisites</h3>
+
+<p>Before installing, make sure you have the appropriate dependencies installed. First, there is OCaml:
+
+<ul>
+    <li>ocaml 4.07.0</li>
```

Comment:
We should specify here whether this specific version of ocaml is needed (and if not, which versions we support).  And also instructions for installing it, or at least a pointer to the website and a generic statement like "you can install ocaml through your system's package manager and install the correct switch with this command."